pytest_splunk_addon_ui_smartx/base_test.py

```diff
@@ -291,11 +291,45 @@ def get_sauce_chrome_opts(self, browser_version):
         return chrome_opts
 
     def get_sauce_safari_opts(self, browser_version):
+        try:
+            retries = 0
+            while retries < 3:
+                response = requests.get("https://api.us-west-1.saucelabs.com/rest/v1/info/platforms/webdriver", verify=False)
+                if response.status_code != 200:
+                    print("Error retrieving supported webdrivers for saucelabs, retrying...")
```

Comment:
@seanmalloy-crest can we have debug logs instead of print statements, and tests appropriate scenarios.

Reply:
This has been changed to log statements as of ecc3f9760ecfac95a66193affd7bf3d039dff0c9